spec/import_spec.rb

```diff
@@ -300,6 +301,32 @@ def upload_file!(name, ext='csv')
         end
       end
 
+      context "with invalid data insert on model validation" do
+        let(:options) { { validate: true } }
+
+        before do
+          Author.create!(name: "John", last_name: "Doe")
+        end
+
+        it "should render both successful and failed message" do
+          upload_file!(:authors_invalid_model)
+          expect(page).to have_content "Failed to import 1 author"
+          expect(page).to have_content "Successfully imported 1 author"
+          expect(Author.count).to eq(2)
+        end
+
+        context "use batch_transaction to make transaction work on model validation" do
+          let(:options) { { validate: true, batch_transaction: true } }
+
+          it "should render only the failed message" do
+            upload_file!(:authors_invalid_model)
+            expect(page).to     have_content "Failed to import 1 author"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.